src/operator/tensor/init_op.h

```diff
@@ -588,6 +622,28 @@ inline bool LinspaceShape(const nnvm::NodeAttrs& attrs,
   return true;
 }
 
+inline bool RangeLikeShape(const nnvm::NodeAttrs& attrs,
+                           mxnet::ShapeVector *in_attrs,
+                           mxnet::ShapeVector *out_attrs) {
+  const RangeLikeParam& param = nnvm::get<RangeLikeParam>(attrs.parsed);
```

Comment:
Could you add a check so that the inferred shape based on start, step and repeat is consistent with the shape inferred by axis?

Reply:
To avoid conflict, I removed the `stop` parameter from `RangLikeParam`.